src/node/redeem.rs

```diff
@@ -175,6 +175,18 @@ impl<J: Jet> RedeemNode<J> {
         self.data.bounds
     }
 
+    pub fn new(
+        arrow: FinalArrow,
+        inner: Inner<Arc<Self>, J, Arc<Value>>,
+        inner_cached: Inner<&Arc<RedeemData<J>>, J, Arc<Value>>,
+    ) -> Result<Self, types::Error> {
+        Ok(RedeemNode {
+            cmr: Cmr::unit(),
+            inner,
+            data: Arc::new(RedeemData::new(arrow, inner_cached)),
+        })
+    }
```

Comment:
4143ea8f374f1633e3c5077f2df25f1bd2d04d2d: I feel this should be removed along with `core::redeem::RedeemNode::to_node()` when we get to that. A node with unit CMR looks weird.

Reply:
:eyes: lol great catch. Indeed, this funciton is only ever used by the temporary conversion code, and I had totally forgotten it existed (and forgotten to remove it in my later commit which deletes `core`).

I'm kinda surprised this passes all tests...it looks like, almost by accident, I only ever used it during encoding and for the policy tests, none of which check CMRs.

Anyway I think it's fine to merge, since it's only needed to shim tests, and we will delete it soon.